code/drasil-example/Drasil/DblPendulum/Concepts.hs

```diff
@@ -9,10 +9,27 @@ pendulum :: CI
 pendulum = commonIdeaWithDict "pendulum" (pn "Pendulum") "Pendulum" [physics]
 
 concepts :: [IdeaDict]
-concepts = [nw rod] 
+concepts = map nw [rod, horizontal, vertical] ++ map nw defs 
        
-rod :: NamedChunk
+rod, horizontal, vertical :: NamedChunk
 rod = nc "rod" (cn' "rod")
+horizontal = nc "horizontal" (cn "horizontal") 
+vertical = nc "vertical" (cn "vertical") 
+---
+
+defs:: [ConceptChunk]
+defs = [shm, amplitude, component, arcLen, motion]
+
+shm, amplitude, component, arcLen, motion :: ConceptChunk
+shm = dcc "SHM" (nounPhraseSP "simple harmonic motion") ("repetitive movement back and forth through" ++
```

Comment:
I think we  can improve a bit on this definition.  SHM still applies when there is damping and in this case the maximum displacements on each side won't be identical over time.  On of the displacements on each oscillation will be less than the other.  I've looked at some on-line definitions, and I like the ones that mention period motion and the notion of an equilibrium position.  How about defining SHM as:

"Periodic motion through an equilibrium position.  The motion is sinusoidal in time and demonstrates a single resonant frequency."

The definition is based on Wikipedia.  Is there a way to cite this source, at least as a comment in our code?

Reply:
I think the home for this knowledge would be in `Physics`.  We probably will want to subdivide physics at some point, but this is a good home for now.